geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloRunner.scala

```diff
@@ -28,15 +28,15 @@ object AccumuloRunner extends Runner {
     new DeleteCatalogCommand(jc),
     new DeleteFeaturesCommand(jc),
     new DeleteRasterCommand(jc),
-    new DescribeCommand(jc),
+    new AccumuloDescribeCommand(jc),
```

Comment:
In pr #1160 this is changing to GetSchemaCommand. Might want to merge master after that pr so you can resolve that.